apps/examples/src/ExamplePage.tsx

```diff
@@ -39,41 +40,69 @@ export function ExamplePage({
 		return () => cancelAnimationFrame(frame)
 	}, [example])
 
+	const categories = examples.map((e) => e.id)
+
 	return (
 		<div className="example">
 			<div className="example__info">
 				<Link className="example__logo" to="/">
 					<ExamplesTldrawLogo /> examples
 				</Link>
-				<ul className="example__info__list scroll-light" ref={scrollElRef}>
-					{examples
-						.filter((e) => !e.hide)
-						.filter((e) => e.order !== null)
-						.map((e) => (
-							<ListLink
-								key={e.path}
-								ref={e.path === example.path ? activeElRef : undefined}
-								example={e}
-								isActive={e.path === example.path}
-							/>
-						))}
-					<li>
-						<hr />
-					</li>
-					{examples
-						.filter((e) => !e.hide)
-						.filter((e) => e.order === null)
-						.map((e) => (
-							<ListLink
-								key={e.path}
-								ref={e.path === example.path ? activeElRef : undefined}
-								example={e}
-								isActive={e.path === example.path}
-							/>
-						))}
-				</ul>
+				<Accordion.Root
+					type="multiple"
+					defaultValue={categories}
+					className="example__info__list scroll-light"
+					ref={scrollElRef}
+				>
+					{categories.map((c) => (
+						<Accordion.Item key={c} value={c}>
+							<Accordion.Trigger className="accordion__trigger">
+								<div className="examples__list__item accordion__trigger__container">
+									<h3 className="accordion__trigger__heading">{c}</h3>
+									<Chevron />
+								</div>
+							</Accordion.Trigger>
+							<Accordion.Content className="accordion__content">
+								<span className="accordion__content__separator"></span>
+								<div className="accordion__content__examples">
+									{examples
+										.find((e) => e.id === c)
```

Comment:
nit: i know this was like this before but it'd be better to avoid a short name like `e` because it can better to avoid shortnames to improve readability. so maybe something like `sidebarExample`? (open to other ideas)